electricpy/__init__.py

```diff
@@ -4438,24 +4438,29 @@ def secondary(val, Np, Ns=1, invert=False):
     return (val * Ns / Np)
 
 
-def tap_changing_transformer(v1, v2, p, q, r, x):
+def tap_changing_transformer(Vgen, Vdis, Pload, Qload, R, X):
     '''
         calulating turn ration for on load tap chaining transformer
```

Comment:
Consider title-casing the docstring heading as `Calculate Turn Ratio of Load Tap Changing Transformer` for consistency with other functions.